test/testdata/lsp/fast_path/abstract_method__1.1.rbupdate

```diff
@@ -0,0 +1,13 @@
+# typed: true
+# assert-fast-path: abstract_method__1.rb,abstract_method__2.rb
+# spacer for exclude from file update
```

Comment:
Total nit: "spacer for exclude-from-file-update"?  (And likewise everywhere else?)

Reply:
Sure, I can update all of these tests in a follow up PR